docs/whats_new.rst

```diff
@@ -35,6 +35,14 @@ Changelog
 ~~~~~~~~~
 - Changed RANSAC chunking logic to reduce max memory use and prefer equal chunk sizes where possible, by `Austin Hurst`_ (`#44 <https://github.com/sappelhoff/pyprep/pull/44>`_)
 
+Bug
+~~~
+- Fixed RANSAC to give consistent results with a fixed seed across different chunk sizes, by `Austin Hurst`_ and `Yorguin Mantilla`_ (`#43 <https://github.com/sappelhoff/pyprep/pull/43>`_)
+
+API
+~~~
+- Added new function :func:`utils.verify_free_ram` and removed `n_samples` argument from the :meth:`NoisyChannels.ransac_correlations`, :meth:`NoisyChannels.run_ransac`, and :meth:`NoisyChannels.get_ransac_pred` methods, by `Austin Hurst`_ and `Yorguin Mantilla`_ (`#43 <https://github.com/sappelhoff/pyprep/pull/43>`_)
```

Comment:
What we are writing in the API docs is meant for users, so if you write that there is a new function `verify_free_ram` you build some kind of expectation that users can/should use this, and that the API is relatively stable. And finally, that it's documented in the API section of the docs.

I suggest we keep `verify_free_ram` private for now, because users do not need to interact with it. Technically that means that we could/should rename the function to start with a leading underscore to follow python convention: `_verify_free_ram`, but it'd be okay with me to skip that for now.

However I recommend to remove that sentence from the user-facing changelog. The exact parameters that changed for the class methods are a bit too much I would say - I think it'd suffice to have a sentence like this:

```suggestion
- The permissible parameters for the following methods were removed and/or reordered: :meth:`NoisyChannels.ransac_correlations`, :meth:`NoisyChannels.run_ransac`, and :meth:`NoisyChannels.get_ransac_pred`, by `Austin Hurst`_ and `Yorguin Mantilla`_ (`#43 <https://github.com/sappelhoff/pyprep/pull/43>`_)
```